dom.go

```diff
@@ -80,7 +80,8 @@ type HTML struct {
 	Node *js.Object
 
 	tag, text, innerHTML   string
-	styles, dataset        map[string]string
+	styles                 map[string]map[string]bool
```

Comment:
Why mo just use `map[string]ClassMap`? it looks like ClassMap can handle multiple classes already! 

I was also thinking about something like this!